project_resource_calendar/views/equipment_calendar_view.xml

```diff
@@ -25,34 +29,37 @@
             <xpath expr="//field[@name='partner_ids']" position="attributes">
                 <attribute name="attrs">{'readonly': [('is_task_event','=',True)]}</attribute>
             </xpath>
+
             <xpath expr="//div[@class='oe_title']" position="inside">
                 <label for="resource_ids" class="oe_edit_only" invisible="1"/>
                 <h2>
                     <field name="resource_ids"
                            widget="many2many_tags"
                            placeholder="Select Resources"
-                           class="oe_inline" 
+                           class="oe_inline"
                            invisible="1" attrs="{'readonly': [('is_task_event','=',True)]}"/>
                 </h2>
                 <label for="room_id" class="oe_edit_only"/>
                 <h2>
-                <field name="room_id"
-                       placeholder="Select Room"
-                       class="oe_inline" attrs="{'readonly': [('is_task_event','=',True)]}"/>
+                    <field name="room_id"
+                           placeholder="Select Room"
+                           class="oe_inline"/>
                 </h2>
                 <label for="equipment_ids" class="oe_edit_only"/>
                 <h2>
-                <field name="equipment_ids" 
-                       widget="many2many_tags"
-                       placeholder="Select Equipment"
-                       class="oe_inline" attrs="{'readonly': [('is_task_event','=',True)]}"/>
+                    <field name="equipment_ids"
```

Comment:
You should keep the readonly attributes for all the fields

Reply:
@rimbendhaou Done